understanding/timing-adjustable.html

```diff
@@ -94,10 +94,10 @@ <h1>達成基準 2.2.1: タイミング調整可能を理解する</h1>
          <section id="intent">
             <h2>意図</h2>
             <p>この達成基準の意図は、障害のある利用者がウェブコンテンツを操作するのに十分な時間の提供を、可能な限り保証することである。全盲、ロービジョン、巧緻性障害、及び、認知能力の低下している利用者は、コンテンツを読んだり、オンラインフォームに記入したりするような操作を実行するのに、より長い時間を必要とする場合がある。ウェブコンテンツの機能が時間の経過に依存している場合、制限時間内に必要な操作を実行することが困難な利用者もいるだろう。このことは、サービスをそういった利用者に対してアクセシブルではないものにしてしまう。そこで、機能を時間の経過に依存しないように設計することで、障害のある利用者がその操作を完了させやすくなる。制限時間を解除する、制限時間の長さを調整する、又は時間切れになる前に制限時間の延長を要求する、といった選択肢を提供することは、作業を無事に終えると見込まれているよりも多くの時間を必要とする利用者の助けになる。これらの選択肢は、利用者にとって最も有用なものから順に書かれている。時間切れになる前に制限時間の延長を要求できるようにすることよりも、制限時間の長さを調整できるようにすることのほうが望ましく、さらにそれよりも制限時間を解除できるようにすることのほうが望ましい。</p>
-            <p>利用者によって開始されることがなく、設定時間後又は定期的に発生する処理は、どれも制限時間によるものである。これには、コンテンツの一部又は全部の更新 (例えば、ページのリフレッシュ)、コンテンツの変更、入力の要求に利用者が対応するためのウィンドウの失効などが含まれる。</p>
-            <p>利用者が読んだり理解したりすることができない速度で進むあるいは更新されるコンテンツも、あわせて含まれる。言い換えれば、アニメーションのある、動きのある、又はスクロールするコンテンツは、利用者がコンテンツを読む能力に対して制限時間を課することになる。</p>
-            <p>この達成基準は、コンテンツが繰り返される又は他のコンテンツと同期する場合、情報及びデータが調整可能であったり利用者の制御下にある限りは、一般に適用されない。この達成基準が適用されない制限時間の例としては、繰り返しスクロールするテキスト、キャプション、及び<a href="http://https://www.w3.org/WAI/tutorials/carousels/">カルーセル</a>が含まれる。これらは制限時間を含むシチュエーションではあるものの、コンテンツが<a href="pause-stop-hide.html">達成基準 2.2.2 一時停止、停止、非表示</a>で規定するようなアクセスするためのコントロールを持っていれば、利用者には利用可能である。
-            <p>なお場合によっては、制限時間を変更することが不可能なことがあり (例えば、オークション又は他のリアルタイムのイベントにおいて)、このため、それらの場合に対する例外規定が提供されている。</p>
+            <p>利用者によって開始されることがなく、設定時間後又は定期的に発生する処理は、どれも制限時間によるものである。これには、コンテンツの一部又は全部の更新 (例えば、ページのリフレッシュ)、コンテンツの変更、入力の要求に利用者が対応するためのウィンドウの期限切れなどが含まれる。</p>
+            <p>利用者が読む及び／又は理解することができない速度で進むあるいは更新されるコンテンツも、あわせて含まれる。言い換えれば、アニメーションのある、動きのある、又はスクロールするコンテンツは、利用者がコンテンツを読む能力に対して制限時間を課することになる。</p>
+            <p>この達成基準は、コンテンツが繰り返される又は他のコンテンツと同期する場合、情報及びデータが調整可能あるいは利用者の制御下にある限りは、一般に適用されない。この達成基準が適用されない制限時間の例としては、繰り返しスクロールするテキスト、キャプション、及び<a href="http://https://www.w3.org/WAI/tutorials/carousels/">カルーセル</a>が含まれる。これらは制限時間を含む状況ではあるものの、コンテンツには<a href="pause-stop-hide.html">達成基準 2.2.2 一時停止、停止、非表示</a>で規定されているようなアクセスするためのコントロールがあるため、利用者にとっては依然として利用可能である。
```

Comment:
> あるいは

とするのであれば「もしくは」になりますね

Reply:
こちらは「又は」ではなく「もしくは」でよいでしょうか…？

Reply:
@momdo レビューありがとうございます。

原文は以下ですが :

> This success criterion is generally not applicable when the content repeats or is synchronized with other content, so long as the information and data is adjustable or otherwise under the control of the end user. 

構造としては下記のようになると思います。

This success criterion is generally not applicable when (the content [repeats] **or** [is synchronized]) with other content, so long as (the information and data) is ([adjustable] **or** [otherwise under the control of the end user]). 

したがって、訳としては :

> この達成基準は、コンテンツが繰り返される又は他のコンテンツと同期する場合、情報及びデータが調整可能又は利用者の制御下にある限りは、一般に適用されない。

...でいかがでしょうか。

Reply:
or otherwise
https://eow.alc.co.jp/search?q=or+otherwise
あるいはその反対で、または［ないしは］別の方法で、～かそうでないもの

とあり、otherwiseを訳に盛り込む方がよいのでは…？という気がします。

Reply:
otherwise の訳出ですね。たしかに。

> この達成基準は、コンテンツが繰り返される又は他のコンテンツと同期する場合、情報及びデータが調整可能**あるいはそうでなくとも**利用者の制御下にある限り、一般に適用されない。

...はいかがでしょうか？

Reply:
> (the information and data) is ([adjustable] or [otherwise under the control of the end user]).

という関係性が成り立っている以上、orは、JISのルールに準じると**又は**か**もしくは**のいずれかしかないので、

情報及びデータが調整可能である、又はそうでなくても利用者の制御下にある限り、

か

情報及びデータが調整可能である、又は別の方法で利用者の制御下にある限り、

となるのかなと思いますがどうでしょうか